indico/migrations/versions/201703141053_447e81273464_added_publish_number_of_registrations_.py

```diff
@@ -0,0 +1,24 @@
+"""added publish_number_of_registrations field to registartion form
```

Comment:
Minor thing, but I'd remove the trailing `_` from the filename and replace this line with the following one (more concise, present tense, fixes typo):

> Add publish_number_of_registrations to registration forms